packages/logger/tests/unit/Logger.test.ts

```diff
@@ -169,7 +203,7 @@ describe('Class: Logger', () => {
 
           // Prepare
           const logger: Logger = createLogger({
-            logLevel: 'ERROR',
+            logLevel: 'SILENT',
```

Comment:
I changed to the highest log level when it doesn't print to stdout from any method and still prints when the sample rate is 1.

Reply:
Could you please elaborate on this one? Not sure I'm following

Reply:
This test case consists of two tests: testing with and w/o sample rate.

For the first one, it is needed to show that no logs emit with the higher log level. The `error` log level was used to show that it doesn't print to stdout from all methods and is ignored on the `error` method:
```
expect(consoleSpy).toBeCalledTimes(method === 'error' ? 1 : 0)
```
Now we have `silence` as the highest level, so I changed to it.

The second one test that all logs emit with sample rate for every method even with the highest log level.

I pointed out this, because this is a dynamic set of tests, and it wasn't obvious to me what is "higher" level means in the description, and why it uses a particular log level. So I thought it worth mentioning in the review.